dynamic_programming/longest_common_subsequence/longestcommonsubsequence.go

```diff
@@ -22,7 +24,7 @@ func LongestCommonSubsequence(a string, b string, m int, n int) int {
 			} else if a[i-1] == b[j-1] {
 				lcs[i][j] = lcs[i-1][j-1] + 1
 			} else {
-				lcs[i][j] = Max(lcs[i-1][j], lcs[i][j-1])
+				lcs[i][j] = knapsack.Max(lcs[i-1][j], lcs[i][j-1])
```

Comment:
Why is Max declared in knapsack? Isn't there any built-in function available?

Reply:
Oh, I used GoLand Ide, It will happen while I created folders for the files inside dynamic programming I guess LongestCommonSubsequence is used the Max function inside the knapsack folder so each of them has their name package inside dynamic_programming to mention to the Max function my Goland automatically added "github.com/TheAlgorithms/Go/dynamic_programming/knapsack" and changed Max to the knapsack.Max

Reply:
Nonetheless, isn't there a built-in max function in standard libraries? I don't see why we need to implement such a basic function ourselves

Reply:
@siriak there is a [max](https://pkg.go.dev/math@go1.17#Max) function in standard library. A lot of the code within this repo creates (and recreated) the max function a few times, one of the reasons why I raised #319 issue.